packages/blocks/github-pr-overview/package.json

```diff
@@ -10,7 +10,7 @@
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/hashintel/hash.git",
+    "url": "https://github.com/hashintel/hash.git#vu/gh-block-improvements",
```

Comment:
It will get changed prior to merging. Added it there so it's easy to test the block on BP Hub => https://github.com/blockprotocol/blockprotocol/pull/407

Reply:
Makes sense, leaving this open so we can resolve just before merging